internal/mesh/internal/controllers/gatewayproxy/controller.go

```diff
@@ -60,9 +68,8 @@ func (r *reconciler) Reconcile(ctx context.Context, rt controller.Runtime, req c
 	}
 
 	if workload == nil {
-		// If workload has been deleted, then return as ProxyStateTemplate should be cleaned up
-		// by the garbage collector because of the owner reference.
-		rt.Logger.Trace("workload doesn't exist; skipping reconciliation", "workload", workloadID)
+		rt.Logger.Trace("workload doesn't exist; skipping reconciliation")
```

Comment:
What's the reason for dropping the workloadID from the log here?

Reply:
honestly this was a blind copy paste from the original, we can add this back